src/App.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
-import './App.css';
-
-const App: React.FC = () => {
-  const projectName = "My Awesome Project";
-  const greeting = "Welcome to my project!";
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import AboutPage from './Pages/AboutPage';
+import UserRegistrationPage from './Pages/UserRegistrationPage';
+import UserAuthorizationPage from './Pages/UserAuthorizationPage';
+import ListOfUsersPage from './Pages/UsersListPage';
+import UserProfilePage from './Pages/UserProfilerPage';
+import ListOfCompaniesPage from './Pages/CompaniesListPage';
+import CompanyProfilePage from './Pages/CompanyProfilePage';
+import WelcomePage from './Pages/WelcomePage';
 
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <h1>{projectName}</h1>
-        <p>{greeting}</p>
-      </header>
-    </div>
+    <Router>
+      <Routes>
+        <Route path="/" element={<WelcomePage />} />
+        <Route path="/about" element={<AboutPage />} />
```

Comment:
export const routerKeys = {
  root: "/",
  about: "/about",
};
etc..

create a router keys object with the all routes you have